week-9/Homework/mandatory/2-exercises/3-atm.js

```diff
@@ -12,7 +12,28 @@
  */
 
 class ATM {
-    // Add your code here
+    constructor() {
+        this.balance = 100;
+    }
+    make_deposit(amount) {
+        if (amount > 0)
+            this.balance += amount;
+        else
+            console.log("you can not add nagative number,please put positive number");
```

Comment:
good addition to validation 
Other checks will be, data type, referencing right functions, ensuring number in a specific format (digits or not)